Wombatlite low o2conditions (#104)
ACCESS-NRI/GFDL-generic-tracers: documentation/docs/pages/Model_description/WOMBATlite_model_description.md

The following are logical statements within the `input.nml` namelist file that can be switched to TRUE or FALSE at runtime. 

| Logical                        | Description                                                                     | Default setting  |
|--------------------------------|---------------------------------------------------------------------------------|------------------|
| `do_caco3_dynamics`            | Production and dissolution of CaCO3 depends on carbon system state              | .true.           |
| `do_colloidal_shunt`           | Fraction of dissolved iron is colloids that coagulate onto sinking material     | .true.           |
| `do_two_ligands`               | Complex soluble iron using two ligands (weak + strong) rather than one          | .false.          |
| `do_burial`                    | Permanently bury a fraction of sinking detrital material into the sediments     | .true.           |
| `do_nitrogen_fixation`         | Allow implicit nitrogen fixing popoulation to add NO<sub>3</sub> to ocean       | .true.           |

> | `do_benthic_denitrification`   | Allow implicit benthic bacterial population to remove NO<sub>3</sub> from ocean | .true.           |

Reviewer: ```suggestion
| `do_burial`                    | Permanently bury a fraction of sinking detrital material into the sediments     | .false.          |
| `do_nitrogen_fixation`         | Allow implicit nitrogen fixing popoulation to add NO<sub>3</sub> to ocean       | .false.          |
| `do_benthic_denitrification`   | Allow implicit benthic bacterial population to remove NO<sub>3</sub> from ocean | .false.          |
```

Author: yes i see why this would be wise... best to maintain N as a closed cycle by default.

> | `do_benthic_denitrification`   | Allow implicit benthic bacterial population to remove NO<sub>3</sub> from ocean | .false.          |

Reviewer: These default to true in the code

```suggestion
| `do_burial`                    | Permanently bury a fraction of sinking detrital material into the sediments     | .true.           |
| `do_nitrogen_fixation`         | Allow implicit nitrogen fixing popoulation to add NO<sub>3</sub> to ocean       | .true.           |
| `do_benthic_denitrification`   | Allow implicit benthic bacterial population to remove NO<sub>3</sub> from ocean | .true.           |
```

Author: Ahh yes thank you.

| `phy_lnit`    | Limitation of phytoplankton by nitrogen                                | dimensionless                                    |
| `phy_lfer`    | Limitation of phytoplankton by iron                                    | dimensionless                                    |
| `phy_dfeupt`  | Uptake of dFe by phytoplankton                                         | mol kg<sup>-1</sup> s<sup>-1</sup>               |
| `tri_mumax`   | Maximum growth rate of trichodesmium                                   | s<sup>-1</sup>                                   |
| `tri_lpar`    | Limitation of trichodesmium by light                                   | dimensionless                                    |

> | `tri_lfer`    | Limitation of trichodesmkum by iron                                    | dimensionless                                    |

Reviewer: ```suggestion
| `tri_lfer`    | Limitation of trichodesmium by iron                                    | dimensionless                                    |
```

| `phylmor`          | Phytoplankton linear mortality rate constant [s⁻¹]                          | 0.005/86400.0      |
| `phyqmor`          | Phytoplankton quadratic mortality rate constant [(mmolC/m³)⁻¹ s⁻¹]          | 0.05/86400.0       |
| `phybiot`          | Phytoplankton biomass threshold [mmolC/m³]                                  | 0.6                |
| `alphabio_tri`     | Trichodesmium initial slope of P–I curve [(W/m²)⁻¹ (mg/mg)⁻¹]               | 1.8                |
| `trikf`            | Trichodesmium half saturation constant for iron uptake [µmolFe/m³]          | 0.125              |

> | `trichlc`          | Trichodesmium chlorophyll to carbon ratio [mol C (mol C)<sup>-1</sup>]      | 0.01               |

Reviewer: ```suggestion
| `trichlc`          | Trichodesmium chlorophyll to carbon ratio [mol Chl (mol C)<sup>-1</sup>]    | 0.01               |
```

- $L_{diazo}^{PAR}$ is the limitation term of diazotroph growth on light (`tri_lpar(i,j,k)`, [dimensionless])  <br>
- $R_{diazo}^{N:C}$ is the ratio of N:C within diazotrophic biomass (`trin2c`, [mol N (mol C)<sup>-1</sup>]) <br>
- $1 \times 10^{-6}$ is a conversion factor to mol kg<sup>-1</sup> <br>

The temperature-dependent maximum growth rate ($\mu_{diazo}^{max}$) is taken directly from [Wrightson et al. (2022)]( https://doi.org/10.1111/gcb.16399) who based their formulation on the work of [Jiang et al. (2018)](https://doi.org/10.1038/s41558-018-021):

$$
\begin{align}
\mu_{diazo}^{max} =& \quad \left( -0.000399(T)^{3} + 0.02685(T)^{2} - 0.555T + 3.633 \right) \dfrac{1}{86400}
\end{align}
$$

_where_ <br>
- $T$ is in situ water temperature (`Temp(i,j,k)`, [ºC]) and we only consider $T > 15.8$ºC <br>
- $\dfrac{1}{86400}$ converts their formula from units of [day<sup>-1</sup>] to [s<sup>-1</sup>] <br>

The iron and light limitation terms are as follows:

$$
\begin{align}
L_{diazo}^{Fe} =& \quad \dfrac{dFe}{dFe + K_{diazo}^{Fe}} \\
L_{diazo}^{PAR} =& \quad 1 - e^{- \alpha_{diazo} PAR}
\end{align}
$$

_where_ <br>

> - $dFe$ is the in situ concentration of dissolved iron (`fe_umolm3`, [nmol Fe kg<sup>-1</sup>]) <br>

Reviewer: ```suggestion
- $dFe$ is the in situ concentration of dissolved iron (`fe_umolm3`, [µmol Fe kg<sup>-1</sup>]) <br>
```

Author: Not the case.

> - $K_{diazo}^{Fe}$ is the half-saturation coefficient for uptake of dissolved iron by diazotrophs (`trikf`, [nmol Fe kg<sup>-1</sup>]) <br>

Reviewer: ```suggestion
- $K_{diazo}^{Fe}$ is the half-saturation coefficient for uptake of dissolved iron by diazotrophs (`trikf`, [µmol Fe kg<sup>-1</sup>]) <br>
```

Author: see above comment.
